### ISSUE TITLE
[Documentation] Broken link to Tracing module

### ISSUE THREAD

**Issue by alexandrnikitin** [twitter/finagle#333]
Broken link to `Tracing` module on the http://twitter.github.io/finagle/guide/Servers.html page. Link points to https://github.com/twitter/finagle/blob/master/finagle-core/src/main/scala/com/twitter/finagle/tracing/TracingFilter.scal which doesn't exist

**Comment by mosesn** [comment]
Yeah, looks like it should link here instead: https://github.com/twitter/finagle/blob/master/finagle-core/src/main/scala/com/twitter/finagle/tracing/TraceInitializerFilter.scala.  Good catch!  Would you like to make a PR?

**Comment by alexandrnikitin** [comment]
With pleasure :bowtie:
Pushed to https://github.com/alexandrnikitin/finagle/tree/issue333_docs_brokenlink
But I get the following error while doing `unidoc` step, not sure what's it:
```
[info] Generating Scala API documentation for main sources to C:\Users\a.nikitin\Documents\Projects\
contribution\finagle\target\scala-2.10\unidoc...
[error] C:\Users\a.nikitin\Documents\Projects\contribution\finagle\finagle-kestrelx\target\scala-2.1
0\src_managed\main\com\twitter\finagle\kestrel\net\lag\kestrel\thriftscala\Item.scala:23: Item is al
ready defined as object Item
[error] object Item extends ThriftStructCodec3[Item] {
[error]        ^
```

**Comment by mosesn** [comment]
Hmm, can you try this: `./sbt 'project finagle-doc' make-site`?  We only need to run unidoc for generating scaladoc–it might be a windows compat problem, since we mostly develop on OSX.

**Comment by alexandrnikitin** [comment]
Fixed by #334
